packages/manager/src/components/DismissibleBanner/DismissibleBanner.tsx

```diff
@@ -88,19 +88,61 @@ export const DismissibleBanner = (props: Props) => {
 
   return (
     <Notice bgcolor={(theme) => theme.palette.background.paper} {...rest}>
-      <Stack direction="row">
-        <Stack direction="column" flex={1} justifyContent="center">
-          {children}
-        </Stack>
+      <Stack direction="row" spacing={1}>
         <Stack
-          alignSelf="flex-start"
-          direction="row"
-          justifyContent="flex-end"
-          spacing={1}
+          flex={1}
+          justifyContent="space-between"
+          sx={{
+            alignItems: { sm: 'center' },
+            flexDirection: {
+              xs: 'column',
+              sm: 'row',
+            },
+            ml: {
+              xs: '8px',
+              sm: '0px',
+            },
+          }}
         >
-          {actionButton}
-          {dismissible ? dismissibleButton : null}
+          <Stack direction="column" flex={1} justifyContent="center">
+            {children}
+          </Stack>
+          <Stack
+            direction="row"
+            spacing={1}
+            sx={(theme) => ({
+              alignItems: {
+                sm: 'center',
+              },
+              paddingLeft: {
+                sm: 4,
+              },
+              mt: {
+                xs: theme.spacingFunction(12),
+                sm: theme.spacingFunction(0),
+              },
+              ml: {
+                xs: theme.spacingFunction(0),
+                sm: theme.spacingFunction(12),
+                md: theme.spacing(24),
+              },
+            })}
+          >
+            {actionButton}
+          </Stack>
         </Stack>
+        {dismissible && (
+          <Stack
+            sx={{
+              alignSelf: {
+                sm: 'center',
+                xs: 'flex-start',
+              },
+            }}
+          >
+            {dismissible ? dismissibleButton : null}
+          </Stack>
+        )}
```

Comment:
Do we need to be checking for `dismissible` twice?  Should this become

```suggestion
        {dismissible && (
          <Stack
            sx={{
              alignSelf: {
                sm: 'center',
                xs: 'flex-start',
              },
            }}
          >
            {dismissibleButton}
          </Stack>
        )}
```

Reply:
++

(What's the conventional commit version of that? "thought: I second this"? 🤔 )